pkg/client/client.go

```diff
@@ -439,6 +440,26 @@ func (c *Client) CreateOrUpdateAlertmanager(ctx context.Context, a *monv1.Alertm
 	return errors.Wrap(err, "updating Alertmanager object failed")
 }
 
+func (c *Client) CreateOrUpdateAlertmanagerConfig(ctx context.Context, a *monv1alpha.AlertmanagerConfig) error {
```

Comment:
should this be moved to the `test/` directory since CMO itself doesn't manage AlertmanagerConfig resources?

Reply:
yes, makes sense